From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 8 Oct 2020 18:47:18 +0000
Subject: polkitAgent: Drop separate user icon style

It should use the same style as used elsewhere (like the lock screen),
so stop using a separate style class to pick up the defaults.

(cherry picked from commit 9d5165b1aa5ad11662c9c02cbf161fd7004e3f05)

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3252
Origin: upstream, 3.38.2
---
 data/theme/gnome-shell-sass/widgets/_dialogs.scss | 4 ----
 js/ui/components/polkitAgent.js                   | 1 -
 2 files changed, 5 deletions(-)

diff --git a/data/theme/gnome-shell-sass/widgets/_dialogs.scss b/data/theme/gnome-shell-sass/widgets/_dialogs.scss
index fc77920..0f568b7 100644
--- a/data/theme/gnome-shell-sass/widgets/_dialogs.scss
+++ b/data/theme/gnome-shell-sass/widgets/_dialogs.scss
@@ -135,10 +135,6 @@
   spacing: 8px;
   margin-bottom: 6px;
 
-  .polkit-dialog-user-icon {
-    border-radius: 99px;
-    background-size: contain;
-  }
   .polkit-dialog-user-root-label { color: $warning_color; }
 }
 
diff --git a/js/ui/components/polkitAgent.js b/js/ui/components/polkitAgent.js
index 5515963..ae75f3c 100644
--- a/js/ui/components/polkitAgent.js
+++ b/js/ui/components/polkitAgent.js
@@ -65,7 +65,6 @@ var AuthenticationDialog = GObject.registerClass({
 
         this._userAvatar = new UserWidget.Avatar(this._user, {
             iconSize: DIALOG_ICON_SIZE,
-            styleClass: 'polkit-dialog-user-icon',
         });
         this._userAvatar.x_align = Clutter.ActorAlign.CENTER;
         userBox.add_child(this._userAvatar);
